### DISCUSSION TITLE
Welcome to epub_viewer Discussions!

### DISCUSSION THREAD

**Discussion by JideGuru** [JideGuru/epub_reader#63] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 312654929** [comment]
radium sdk progression

**Reply by JideGuru** [reply]
I made a discussion for this here https://github.com/JideGuru/epub_viewer/discussions/64
We can discuss there
